# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../file/zstd.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur file-5.28/magic/Magdir/compress file-5.28-zstd/magic/Magdir/compress
--- file-5.28/magic/Magdir/compress	2015-12-04 21:48:03.000000000 +0100
+++ file-5.28-zstd/magic/Magdir/compress	2016-10-04 19:50:25.703283246 +0200
@@ -77,6 +77,9 @@
 !:mime	application/x-bzip2
 >3	byte		>47		\b, block size = %c00k
 
+# zstd
+0	long		0xfd2fb528	zstd compressed data
+
 # lzip
 0	string		LZIP		lzip compressed data
 !:mime application/x-lzip
